test/integration/dns_forward_example_multi_names/controls/gcloud.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id       = attribute('project_id')
+addresses        = attribute('addresses')
+dns_fqdns        = attribute('dns_fqdns')
+names            = attribute('names')
+forward_zone     = attribute('forward_zone')
+
+control "dns-forward-example-multi-name" do
```

Comment:
cc @akshaybathija-github who is also upgrading the tests for this module for any thoughts on how to sequence

Reply:
Lets prioritize ones with most usage since we can't do them all at once. @akshaybathija-github - it'll be nice if we could get a backlog going internally to track all conversions.

Reply:
Sure @g-awmalik,  i am currently working on migration of this module as we speak